system/Helpers/number_helper.php

```diff
@@ -242,3 +242,99 @@ function format_number($num, int $precision = 1, string $locale = null, array $o
 }
 
 //--------------------------------------------------------------------
+
+if ( ! function_exists('number_to_roman'))
+{
+	/**
+	* Convert a number to a roman numeral.
+	* 
+	* @param int $num it will convert to int
+	*
+	* @return string
+	*/
+	function number_to_roman($num)
+	{
+		$num = (int) $num;
+		if ($num < 1 OR $num > 3999)
+		{
+				return;
+		}
+		return _number_to_roman($num, 1);
```

Comment:
I'm not a big fan of having two separate methods here. I see that `_number_to_roman` is called recursively, but as little as the main function does, it seems you could merge the two functions and have them still work. Please try to do that then we'll consider it.